docs/docusaurus/docs/cloud/connect/connect_postgresql.md

```diff
@@ -25,28 +23,51 @@ New to GX Cloud and not sure that it's the right solution for your organization?
 
 - You have stopped all local running instances of the GX Agent.
 
-## Prepare your PostgreSQL environment
+## Connect to a PostgreSQL Data Asset
+
+1. In GX Cloud, click **Data Assets** > **New Data Asset**.
+
+2. Copy the code in the code pane.
+
+3. Prepare your PostgreSQL environment:
+
+   - In pgAdmin, select a database.
+
+   - Click **Tools** > **Query Tool**.
+
+   - Paste the code you copied in step 2 into the **Query** pane to create and assign the `gx_role` role and allow GX Cloud to access to all `public` schemas and tables on a specific database.
+
+      Replace `<your_password>` and `<your_database>` with your own values. `ALTER DEFAULT PRIVILEGES IN SCHEMA public GRANT SELECT ON TABLES TO gx_role;` is optional and gives the `gx_role` user access to all future tables in the defined schema.
+
+    - Click **Execute/Refresh**.
+
+4.  In GX Cloud, click **I have created a GX Cloud user with valid permissions** and then click **Continue**.
+
+5. Enter a meaningful name for the Data Asset in the **Data Source name** field.
```

Comment:
This is technically the name for the Data Source. The data asset names are defined later

Reply:
@annabarr I have updated the text.